src/test/hotcuecontrol_test.cpp

```diff
@@ -1092,28 +1092,6 @@ TEST_F(HotcueControlTest, CueLoopWithSavedLoopToggles) {
     EXPECT_TRUE(m_pLoopEnabled->toBool());
 }
 
-TEST_F(HotcueControlTest, CueLoopWithoutLoopOrBeats) {
-    createAndLoadFakeTrack();
-
-    EXPECT_DOUBLE_EQ(static_cast<double>(HotcueControl::Status::Empty), m_pHotcue1Enabled->get());
-    EXPECT_FALSE(mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(
-            m_pHotcue1Position->get())
-                         .isValid());
-    EXPECT_FALSE(mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(
-            m_pHotcue1EndPosition->get())
-                         .isValid());
-
-    m_pHotcue1CueLoop->set(1);
-    m_pHotcue1CueLoop->set(0);
-
-    EXPECT_DOUBLE_EQ(static_cast<double>(HotcueControl::Status::Set), m_pHotcue1Enabled->get());
-    EXPECT_FRAMEPOS_EQ_CONTROL(mixxx::audio::kStartFramePos, m_pHotcue1Position);
-    EXPECT_FALSE(mixxx::audio::FramePos::fromEngineSamplePosMaybeInvalid(
-            m_pHotcue1EndPosition->get())
-                         .isValid());
-    EXPECT_FALSE(m_pLoopEnabled->toBool());
-}
-
```

Comment:
Why did you remove these unit tests? Now we have more cases to test and not less.

Reply:
This tests that there is no (cue) loop created if the track has no beats. This situation will not occur anymore (with a track loaded).

We don't need to test "no beats" cases --> less tests.
What new cases are you referring to?